geoportal/c2cgeoportal_geoportal/scaffolds/update/geoportal/CONST_vars.yaml_tmpl

```diff
@@ -40,6 +40,8 @@ vars:
   authentication:
     two_factor: false
     two_factor_issuer_name: {{project}}
+    oauth2_token_expire_minutes: 60
+    oauth2_authorization_expire_minutes: 10
```

Comment:
Are you sure about these values, I would think the token expire after 10 minutes but the authorization remains for 60 min.

Reply:
It's correct, I use the proposal values from here: https://oauthlib.readthedocs.io/en/latest/oauth2/server.html